bootstraptest/test_yjit.rb

```diff
@@ -1832,6 +1832,18 @@ def jittable_method
   jittable_method
 }
 
+# test getbyte on string class
+assert_equal '[97, nil]', %q{
+  def getbyte(s, i)
+    s.getbyte(i) || :nil
```

Comment:
```suggestion
    s.getbyte(i) || nil
```

Reply:
Thank you! I updated as part of [`38590fe`](https://github.com/ruby/ruby/compare/76251be9e26e088d319600d5c9b895d609cf89ac..38590fe1b9e0bcb3a226390714f9bf4f19011eae)